frontend/src/modules/app/components/app/index.js

```diff
@@ -7,7 +7,7 @@ import { WrappedBooks as Books } from 'modules/app/components/books'
 import { WrappedBook as Book } from 'modules/app/components/book'
 import Chapter from 'modules/app/components/book/chapter'
 import BookNotes from 'modules/app/components/book/notes'
```

Comment:
My flow picked this up as an error because WrappedBooked was being exported not WrappedNote